miden-tx/src/host/mod.rs

```diff
@@ -37,7 +39,7 @@ pub const STORAGE_TREE_DEPTH: Felt = Felt::new(AccountStorage::STORAGE_TREE_DEPT
 // ================================================================================================
 
 /// Transaction host is responsible for handling [Host] requests made by a transaction kernel.
-pub struct TransactionHost<A> {
+pub struct TransactionHost<'a, A> {
```

Comment:
Is it possible to avoid using a lifetime parameter here? (e.g., could we use `&'static` for strings in `kernel_assertion_errors`?)